digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -383,7 +383,7 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
   get("/roadlinks/roadaddress/project/all/projectId/:id") {
     val projectId = params("id").toLong
     try {
-      projectService.getRoadAddressSingleProject(projectId) match {
+      projectService.getRoadAddressSingleProject(projectId, false) match {
```

Comment:
Use like this `projectService.getRoadAddressSingleProject(projectId, withActiveRoadAddress = false)` to be easier to read